rosdep/python.yaml

```diff
@@ -6997,6 +6997,15 @@ python3-pygame:
       pip: [pygame]
     xenial:
       pip: [pygame]
+python3-pygit2:
+  alpine: [py3-pygit2]
```

Comment:
https://pkgs.alpinelinux.org/package/edge/community/x86/py3-pygit2

Reply:
@ivanpauno Is the alpine one incorrect?

Reply:
@ivanpauno Of course, are you sure python-pygit2 for rhel is the python3 variant?  What

Reply:
> @ivanpauno Is the alpine one incorrect?

No, sorry for confusing you.
I only added the link in a comment because I didn't see it in the OP.

Reply:
> @ivanpauno Of course, are you sure python-pygit2 for rhel points to the python3 variant?

Yes, the previous link I copied was wrong though https://centos.pkgs.org/8/epel-aarch64/python3-pygit2-0.26.4-1.el8.aarch64.rpm.html.
You have to use the `python3_pkgversion` macro, as commented above: `python%{python3_pkgversion}-pygit2`.
e.g.: https://github.com/ros/rosdistro/blob/56a6a44dd56e76ec3148bbc6c708757006c954c0/rosdep/python.yaml#L315
See also https://github.com/ros/rosdistro/pull/20842#issue-268956327.

Reply:
@ivanpauno Thanks for the clarification I was not aware of the macro. I will push these changes later tonight.

Reply:
@ivandariojr I force pushed the changes into this pull request so that we still have one commit.